tests/unit/BitrateTest.ts

```diff
@@ -493,6 +503,94 @@ describe('BitrateTest', () => {
           clock.tick(1200);
         });
 
+        describe('ICE Candidate Stats', () => {
+          const runBitrateTest = (shoulStop: boolean) => {
```

Comment:
```suggestion
          const runBitrateTest = (shouldStop: boolean) => {
```

Reply:
I updated a couple of these references :)